src/task.c

```diff
@@ -729,8 +730,10 @@ static char *jl_alloc_fiber(jl_ucontext_t *t, size_t *ssize, jl_task_t *owner)
     char *stkbuf = (char*)jl_malloc_stack(ssize, owner);
     if (stkbuf == NULL)
         return NULL;
+#ifndef __clang_analyzer__
```

Comment:
Curious why this is here?

Reply:
The analyzer gets confused by this syntax construct because it's inside a rooted value. `&ptls->base_ctx`. Could probably be fixed, but didn't seem worth the effort.